### PR TITLE
Provide instructions to build and run Parchment

### PR DESCRIPTION
**Pull request by dfabulich**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,11 +10,61 @@ Included projects
...
+
+```
+npm install
+```
+
+You'll then need to build Emglken. (Emglken requires [Docker](https://www.docker.com/) to build, and it can take several minutes. You'll typically only need to do this once, unless you're hacking on Emglken itself.)
+
+```
+npm run emglken
+```
```
Comment (dfabulich) [inline README.md:39]: We should remove this section if/when #79 merges.

**Comment by dfabulich** [comment]
I substantially rebuilt this PR since much of it has become obsolete.

I've configured a `prepare` script to run the gulp build, which will ensure that we automatically run the gulp build during `npm install`.

In addition, the new http server script will launch a web server _and_ watch the files for changes to automatically rebuild them. (I've renamed the `serve` script to `start`, so you can type `npm start` instead of `npm run serve`.)

Finally, with a fix for #80, we don't have to document how to use the `?story` parameter to test Parchment; users can just submit local files for testing.

File: README.md [hunk 2]
```diff
@@ -3,11 +3,47 @@ Parchment
 
 Parchment is the Interactive Fiction player for the web. To play a story with Parchment go to <https://iplayif.com>!
 
+Building Instructions
+---------------------
+
+The upstream projects are included as git submodules. You'll have to start by initializing the submodules, like this:
+
+```
+git submodule update --init --recursive
```
Comment (curiousdannii) [inline README.md:12]: I don't think recursive is needed now

File: README.md [hunk 3]
```diff
@@ -3,11 +3,47 @@ Parchment
 
 Parchment is the Interactive Fiction player for the web. To play a story with Parchment go to <https://iplayif.com>!
 
+Building Instructions
```
Comment (curiousdannii) [inline README.md:6]: Can this section be moved to the bottom?

File: package.json [hunk 4]
```diff
@@ -33,9 +33,11 @@
   },
   "scripts": {
     "gulp": "gulp",
+    "prepare": "gulp",
```
Comment (curiousdannii) [inline package.json:36]: What's the difference between prepare and prestart? Do we need both? And can you put them in alphabetical order?

**Review by curiousdannii — CHANGES_REQUESTED** [review]

File: package.json (hunk 4 above)
Comment (dfabulich) [reply package.json:36]: https://docs.npmjs.com/cli/v8/using-npm/scripts documents the built-in npm scripts. `npm install` will run `prepare` but `npm start` won't, so adding as `prestart` ensures that we run the `gulp` build before launching the web server and the build watcher.

I'll alphabetize.

File: README.md (hunk 2 above)
Comment (dfabulich) [reply README.md:12]: Eh, it's harmless, and without it, `link-local-emglken` won't work quite right.

**Review by curiousdannii — APPROVED** [review]

**Pull request merged**